src/workers/postAddedSlackChannelSend.ts

```diff
@@ -38,6 +39,24 @@ export const postAddedSlackChannelSendWorker: TypedWorker<'api.v1.post-visible'>
         ]);
         const source = await post.source;
 
+        let adminJoinToken: string | undefined;
```

Comment:
Is it not automatically inferred to be `| undefined` if we don't define a value for it?

Reply:
nope, it just says string